lib/debug-server.js

```diff
@@ -62,8 +63,12 @@ DebugServer.prototype._getDebuggerPort = function(url) {
   return parseInt((/\?port=(\d+)/.exec(url) || [null, this._config.debugPort])[1], 10);
 };
 
-DebugServer.prototype._createSession = function(debugPort) {
-  return Session.create(debugPort, this._config);
+DebugServer.prototype._getDebuggerHost = function(url) {
+  return (/[\?\&]host=([0-9.]+)/.exec(url) || [null, this._config.debugHost])[1];
```

Comment:
This will support only IP addresses, hostnames like `localhost` won't work. Do we care?